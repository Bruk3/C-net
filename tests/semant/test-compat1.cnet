int main() {
    4 + 'a';

    return 0;
}
