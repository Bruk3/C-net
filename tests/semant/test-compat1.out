int main()
{
(int : (int : 4) + (int : a));
}

